crates/sargon-uniffi/src/signing/transaction_sign_request_input.rs

```diff
@@ -38,6 +38,19 @@ macro_rules! decl_transaction_sign_request_input {
             );
         }
     };
+    (signable_and_payload: $signable:ty) => {
+        paste! {
```

Comment:
Please use new macro system - preinterpret. See the matrix macro

Reply:
https://github.com/radixdlt/sargon/blob/main/crates/sargon-uniffi/src/profile/mfa/security_structures/matrices/decl_matrix_macro.rs

Reply:
Rationale: crate `paste` is no longer maintained and is gonna be deprecated.

preinterpret is also better.

Created by @dhedey so ask in Rust channel for help from him

Reply:
But also we have bigger fish to fry... feel free to ignore